.tekton/osc-podvm-builder-push.yaml

```diff
@@ -101,7 +103,7 @@ spec:
       description: Skip checks against built image
       name: skip-checks
       type: string
-    - default: "false"
+    - default: "true"
```

Comment:
Why this specific changes?

Reply:
This is the flag that enables hermetic build (see the description below it).
That's maybe the confusion: the description for the flag is below it, not above ?

Reply:
I hate when the the json has as the first element something random :) I was reading the skip-checks instead of hermetic :)

Reply:
So did I @beraldoleal 😉

Reply:
As a matter of fact, I suggest [rewriting them](https://github.com/openshift/sandboxed-containers-operator/commit/a58d5427520eae434f943757ce9278b1ea46d0e9).

Reply:
I don't mind, but I'm not sure how long it will stay.
This file is initially generated by Konflux, and regularly updated by it (for tekton image references). And I've seen a couple times my refactoring to be reset by updates (typically the "on-cel-expression" at the top of the file).
Maybe it won't touch it, but I wouldn't bet :-)

Reply:
cherry-picked